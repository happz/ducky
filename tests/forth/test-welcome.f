: WELCOME
  ." DuckyFORTH VERSION " VERSION . CR
  ." OK "
;

: TEST
  WELCOME
;

