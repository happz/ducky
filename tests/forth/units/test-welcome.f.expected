DuckyFORTH VERSION 2 
1176 CELLS REMAINING
OK 